.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+Short summary of your changes. Explain what have you done and comment the decisions that you may have taken.
+
+### Related Trello cards:
+* Add a link for each Trello cards related to this PR. Don't forget to update the cards status in Trello.
```

Comment:
I think it's better, because it indicates code to be removed. Maybe we can apply it to other texts.